ps_emailsmanager.php

```diff
@@ -283,6 +307,11 @@ public function saveTemplateConf()
             //Fetch translations from addons api
             $translations = $this->getEmailsTranslations($language['iso_code']);
 
+            $lang_fields_value = $this->getLangFieldsTranslations($lang_fields, $language['id_lang']);
```

Comment:
This function will be deleted and code simplified !
I'm doing it here to assign the value in the right language context before the fetch.